src/main/cpp/WheelDrive.cpp

```diff
@@ -14,100 +13,93 @@ WheelDrive::WheelDrive(int angleMotorPort, int speedMotorPort,
     angleMotor.SetSafetyEnabled(false);
     speedMotor.SetSafetyEnabled(false);
 
-    // m_canCoder.SetStatusFramePeriod(CANCoderStatusFrame::CANCoderStatusFrame_SensorData, 20, 100);
+    // m_canCoder.SetStatusFramePeriod(CANCoderStatusFrame::CANCoderStatusFrame_SensorData,
+    // 20, 100);
     m_canCoder.ClearStickyFaults();
 }
 
-
-//Drive function, set each module's speed and angle
-void 
-WheelDrive::drive(double speed, double angle, double offSet){    
+// Drive function, set each module's speed and angle
+void WheelDrive::drive(double speed, double angle, double offSet) {
     double value = getAngle(offSet);
-    if(abs(value - angle) > 90){
-        angle = (angle > 0 )? angle - 180: angle + 180;
+    if (abs(value - angle) > 90)  // TODO: this doesn't handle wrapping behavior

```

Comment:
Be sure to revisit this code to handle angle wrapping behavior for a more robust swerve.
Imagine `value` is 359 (or -179) and angle is 1 (or 179), this code wouldn't properly measure the distance between those angles.